Components/avatar.css

```diff
@@ -0,0 +1,58 @@
+.component-container{
+    display: flex;
+    flex-flow: row wrap;
+    align-items: baseline;
+    border-radius: 0.3rem;
+    justify-content: center;
+    border: 1px solid var(--dark-secondary-color);
+    background-color: var(--secondary-color);
+}
+  .avatar-lg {
+    width: 60px;
+    height: 60px;
+  }
+  .avatar-x-lg {
+    width: 70px;
+    height: 70px;
+  }
+  .avatar-xx-lg {
+    width: 80px;
+    height: 80px;
+  }
+  .avatar-sm {
+    width: 40px;
+    height: 40px;
+  }
+  .avatar-x-sm {
+    width: 30px;
+    height: 30px;
+  }
+  .avatar-md {
+    width: 50px;
+    height: 50px;
+  }
```

Comment:
Try to use relative units instead of absolute units.

Reply:
Changed this to relative units